contrib/buildahimage/centos7/Dockerfile

```diff
@@ -8,19 +8,32 @@ FROM centos:7
 
 # Remove directories used by yum that are just taking
 # up space.
-RUN useradd build; yum -y update; rpm --restore shadow-utils 2>/dev/null; yum -y install cpp buildah fuse-overlayfs xz; rm -rf /var/cache /var/log/dnf* /var/log/yum.*;
+RUN useradd build && \
+    yum -y update && \
+    yum -y reinstall shadow-utils && \
+    yum -y install buildah fuse-overlayfs xz && \
+    rm -rf /var/cache /var/log/dnf* /var/log/yum.*
 
-ADD https://raw.githubusercontent.com/containers/buildah/main/contrib/buildahimage/stable/containers.conf /etc/containers/
+ARG _REPO_URL=
+ADD $_REPO_URL/storage.conf /etc/containers/storage.conf
```

Comment:
Oh woops, this is my bad.  The file gets overwritten (below) by `sed`.  Fix: https://github.com/containers/podman/pull/14500